Engine/source/math/mathTypes.cpp

```diff
@@ -215,6 +225,31 @@ ConsoleSetType( TypePoint3F )
 }
 
 //-----------------------------------------------------------------------------
+// TypePoint4I
+//-----------------------------------------------------------------------------
+ConsoleType( Point4I, TypePoint4I, Point4I )
+ImplementConsoleTypeCasters( TypePoint4I, Point4I )
+
+ConsoleGetType( TypePoint4I )
+{
+   Point4I *pt = (Point4I *) dptr;
+   static const U32 bufSize = 256;
+   char* returnBuffer = Con::getReturnBuffer(bufSize);
+   dSprintf(returnBuffer, bufSize, "%d %d %d %d", pt->x, pt->y, pt->z, pt->w);
+   return returnBuffer;
+}
+
+ConsoleSetType( TypePoint4I )
+{
+   if(argc == 1)
+      dSscanf(argv[0], "%d %d %d %d", &((Point4I *) dptr)->x, &((Point4I *) dptr)->y, &((Point4I *) dptr)->z, &((Point4I *) dptr)->w);
+   else if(argc == 4)
+      *((Point4I *) dptr) = Point4I(dAtoi(argv[0]), dAtoi(argv[1]), dAtoi(argv[2]), dAtoi(argv[3]));
```

Comment:
Should be noted that argc is always 1 here (this is a case for a console feature which was never implemented), thus this code will never be called.